playwright.config.ts

```
@@ -0,0 +1,43 @@
+    },
+    {
+      name: 'Mobile Chrome',
+      use: { ...devices['Pixel 5'] },
+    },
+    {
+      name: 'Mobile Safari',
+      use: { ...devices['iPhone 12'] },
+    },
+  ],
+  webServer: {
+    command: 'npm run dev',
```

ESLint

[error] Replace `'npm·run·dev'` with `"npm·run·dev"`

(prettier/prettier)